docs/network-drivers.md

```diff
@@ -213,6 +215,50 @@ The body of the api call request will look like:
 The brocade driver accepts interface names the same way they would be accepted
 in the console of the switch, ex. ``101/0/10``.
 
+### Juniper QFX5200 driver
+
+#### Switch preparation
+
+1. The driver relies on netconf protocol to manage the switch. 
+It is important switch is configured for responding to netconf connections. 
+With administration privileges, running the following command in 
+configuration mode should set it up for netconf based connections. 
+
+```
+set system services netconf ssh
+```
+
+2. Driver relies on the PyEZ library, a python wrapper that for netconf 
+using `ncc client`. Make sure it is installed before using the driver. 
+
+3. VLANS that HIL will manage should be preconfigured on switch. 
+
+4. Configure the initial state of the interface (ports) configuration manually.
+Then copy the configuration as a jinja2 template and store it at
+`hil/ext/switches/junos/jinja_templates`. This will used by `revert_port` 
+to reset the ports to its initial conditions when required. 
+
+#### switch_resister
+
+To register a Juniper QFX5200 switch, the ``"type"`` field of the request body
+must have a value of::
+
+	http://schema.massopencloud.org/haas/v0/switches/juniper
+
+In addition, it requires three extra fields: 
+``"username"``, ``"hostname"``, and ``"pasword"``, which provide the necessary
+information to connect to the switch via netconf over ssh using PyEZ library.
+``"hostname"`` can be an IP address too. 
```

Comment:
No it does not have to be a url. In fact it should not be, since pyEZ makes ssh connections. It does not make any rest calls. `hostname` should be either IP address or a hostname or FQDN.